crypto/cmp/cmp_ctx.c

```diff
@@ -36,7 +34,6 @@ X509_STORE *OSSL_CMP_CTX_get0_trustedStore(const OSSL_CMP_CTX *ctx)
  * Set certificate store containing trusted (root) CA certs and possibly CRLs
  * and a cert verification callback function used for CMP server authentication.
  * Any already existing store entry is freed. Given NULL, the entry is reset.
- * returns 1 on success, 0 on error
```

Comment:
Just because I felt these comments are pretty redundant.
Nearly all functions in this file are public and thus fully documented.
The return values are documented in the pod files and obvious in the sources.
Anyway the ones remove here are just the usual `1` for success and `0` for error.